docs/guides/naming-conventions.md

```diff
@@ -18,50 +18,49 @@ kinds of files.
 
 ### Identifiers   
 
-Commands and events reference model entities via their identifiers. 
+Commands and events reference model entities using their identifiers. 
 Having typed identifiers makes a model type safe.
 Although the framework also supports `String`, `Integer`, and `Long` as valid ID types, 
-we strongly recommend defining custom ID types like `CustomerId`, `OrderId`, `ShipmentId`, etc. 
+we strongly recommend defining custom ID types like `CustomerId`, `OrderId`, `ShipmentId`, and others. 
 You can find similar cases in the framework API which has `EventId`, `CommandId`, `UserId`, 
-`TenantId`, etc. 
+`TenantId`, and others. 
 
-We found it convenient to define ID types in one file called `identifiers.proto`. 
-A typical project is likely to have more than one Bounded Context, so you
-are going to have several `identifiers.proto` each residing under the directory with proto
-files defining the data model of the corresponding Bounded Context.   
+We find it convenient to define ID types in one file called `identifiers.proto`. 
+A typical project is likely to have more than one Bounded Context, thus you will have several `identifiers.proto` files. 
+Each of them resides under the directory with proto files defining the data model of the corresponding Bounded Context.   
 
 ### Command definitions
 
-Commands are defined in a file which names end with `commands.proto`. 
-It can be simply `commands.proto`, but usually commands are handled by different entities. 
-So, it's convenient to name such a file after the type of the target entity, 
-for instance, an aggregate: 
+Commands are defined in a file ending with `commands.proto`. 
```

Comment:
We need to mention a name of a file. Otherwise there is a small chance of confusing the name of a file with its content: “Do I put this text at the end of the file?”